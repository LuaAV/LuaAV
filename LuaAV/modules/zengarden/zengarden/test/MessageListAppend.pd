#N canvas 678 22 450 300 10;
#X obj 158 231 list append 1 2;
#X obj 158 268 print append;
#X msg 58 149 1 2 3;
#X obj 114 21 loadbang;
#X obj 194 21 bng 15 250 50 0 empty empty empty 17 7 0 10 -262144 -1
-1;
#X msg 115 149 walk the dog;
#X msg 258 148 go dog go;
#X obj 338 149 b;
#X obj 217 149 b;
#X obj 83 115 t b b b;
#X obj 205 60 t b b b b b;
#X connect 0 0 1 0;
#X connect 2 0 0 0;
#X connect 3 0 10 0;
#X connect 4 0 10 0;
#X connect 5 0 0 0;
#X connect 6 0 0 1;
#X connect 7 0 0 1;
#X connect 8 0 0 0;
#X connect 9 0 2 0;
#X connect 9 1 5 0;
#X connect 9 2 8 0;
#X connect 10 0 9 0;
#X connect 10 1 6 0;
#X connect 10 2 9 0;
#X connect 10 3 7 0;
#X connect 10 4 9 0;
