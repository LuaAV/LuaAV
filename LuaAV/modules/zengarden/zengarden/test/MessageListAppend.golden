[@ 0.000ms] append: 1 2
[@ 0.000ms] append: walk the dog 1 2
[@ 0.000ms] append: 1 2 3 1 2
[@ 0.000ms] append: bang
[@ 0.000ms] append: walk the dog
[@ 0.000ms] append: 1 2 3
[@ 0.000ms] append: go dog go
[@ 0.000ms] append: walk the dog go dog go
[@ 0.000ms] append: 1 2 3 go dog go